test/scripts/helpers/open_graph.js

```diff
@@ -35,9 +35,9 @@ describe('open_graph', () => {
         meta({property: 'og:title', content: hexo.config.title}),
         meta({property: 'og:url'}),
         meta({property: 'og:site_name', content: hexo.config.title}),
+        meta({property: 'og:locale', content: hexo.config.language}),
```

Comment:
duplicate of https://github.com/hexojs/hexo/pull/3660

Reply:
please remove [this line](https://github.com/hexojs/hexo/blob/86316ddba28125cf3cbe3bcdbfb6ba326186891c/test/scripts/helpers/open_graph.js#L435) and update [this line](https://github.com/hexojs/hexo/blob/86316ddba28125cf3cbe3bcdbfb6ba326186891c/test/scripts/helpers/open_graph.js#L427).

Reply:
I will rebase and remove that.